src/lib/parts.ts

```diff
@@ -131,34 +68,12 @@ export class AttributeCommitter {
 
   protected _getValue(): unknown {
     const strings = this.strings;
-    const parts = this.parts;
     const l = strings.length - 1;
-
-    // If we're assigning an attribute via syntax like:
-    //    attr="${foo}"  or  attr=${foo}
-    // but not
-    //    attr="${foo} ${bar}" or attr="${foo} baz"
-    // then we don't want to coerce the attribute value into one long
-    // string. Instead we want to just return the value itself directly,
-    // so that sanitizeDOMValue can get the actual value rather than
-    // String(value)
-    // The exception is if v is an array, in which case we do want to smash
-    // it together into a string without calling String() on the array.
-    //
-    // This also allows trusted values (when using TrustedTypes) being
-    // assigned to DOM sinks without being stringified in the process.
-    if (l === 1 && strings[0] === '' && strings[1] === '' &&
-        parts[0] !== undefined) {
-      const v = parts[0].value;
-      if (!isIterable(v)) {
-        return v;
-      }
-    }
     let text = '';
 
     for (let i = 0; i < l; i++) {
       text += strings[i];
-      const part = parts[i];
```

Comment:
I think we should keep these changes, as they let us pass non-string values straight into the DOM, which is desirable for trusted types (even if we revert the trusted types integration where we .innerHTML)

Reply:
I did revert the trusted types integration too. My feeling is we can easily bring this back, and as long as we're reverting basically everything, we might as well do everything. Keeping this would still be a few extra bytes for no observable difference yet.

Another way of looking at this is that we shouldn't split the cost of a feature across too many changes as that'll obscure the cost.